benchmarks/README.md

```diff
@@ -6,11 +6,11 @@ Example sites for benchmarking `gatsby`.
 
 The standard interface for running a benchmark is:
 
-```
+```shell
 cd {benchmark directory}
 export NUM_PAGES={n}
 npm install
-gatsby build
+npm run build / gatsby build
```

Comment:
I think just having the `gatsby build` command works because using npm run build would depend on if you had npm as your package manager. it would be preferable to keep the command significant to Gatsby's commands. 
```suggestion
 gatsby build
```